packages/theme/example/index.html

```diff
@@ -1020,6 +1020,12 @@ <h1 id="forms">Forms</h1>
                 <div class="form-group has-error">
                   <input type="text" class="form-control" id="inputError">
                   <label class="control-label" for="inputError">Input error</label>
+                  <div>
+                    <p></p>
```

Comment:
Why is there an empty paragraph here?

Reply:
the error div with empty <p> is generated by react, the html is copied from other applications, PFA
![error_div_with_empy_pg](https://user-images.githubusercontent.com/34709414/36558053-28260212-180a-11e8-86b6-4fea5a9ba3ad.png)

Reply:
But here it's a demo page, it's an issue in the generated form, you can remove it

Reply:
removed the error message with empty paragraph